### PR TITLE
Css ecommerce page

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2021-12-10 at 9 57 35 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey DJ, there is too much vertical space between your lines of text as compared to the example, please revise.

**Comment by dcatan89** [comment]
Screen Shot 2021-12-10 at 9 42 27 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Still too much space 👀 
image

**Comment by dcatan89** [comment]
Screen Shot 2021-12-13 at 11 03 31 AM

File: css-ecommerce-page/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row header-arrivals">
+      <h1 class=""> New Arrivals </h1>
+    </div>
+    <div class="row">
+      <div class="column-full">
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html]: This column takes up only one fifth of the width, right? Why is it set as `column-full`? The rest of your columns are the same way, you should revise this to correctly describe its content.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-ecommerce-page/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+              </h4>
+              <h4 class="item-price-section  lv-words-section">
+                <span class="prices font "> $95 </span>
+                <span class="font lgrey-font "><s>$1200 retail </s> </span>
+              </h4>
+        </div>
+      </div>
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right">
+          <div class="columnn-fifth width">
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html]: `width` is just setting this element to be full width, you should use `column-full` for that

File: css-ecommerce-page/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        <div class=" row  flex-column margin-left-right">
+          <div class="lv-bag-img "> </div>
+              <h4 class="item-price-section lv-words-section">
+                <span class="brand font"> Louis Vuitton </span>
+                <span class="sizes-info font lgrey-font">One-size </span>
+              </h4>
+              <h4 class="item-price-section  lv-words-section">
+                <span class="prices font "> $95 </span>
+                <span class="font lgrey-font "><s>$1200 retail </s> </span>
+              </h4>
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html:70]: Your indentation needs to get fixed up here.

File: css-ecommerce-page/index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        </div>
+      </div>
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right ">
+          <div class="columnn-fifth width">
+            <img class="tiffany-key-img " src="images/key.jpeg">
+          </div>
+          <h4 class="item-price-section">
+            <span class="brand font"> Tiffany </span>
+            <span class="sizes-info font  lgrey-font"> One-size</span></h4>
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html]: Please move the closing tag for the `h4` to its own line

File: css-ecommerce-page/index.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+</head>
+<body>
+  <div class="container">
+    <div class="row header-arrivals">
+      <h1 class=""> New Arrivals </h1>
+    </div>
+    <div class="row">
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right">
+          <div class="columnn-fifth width ">
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html]: `width` is just setting this element to be full width, you should use `column-full` for that

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**